lang/ui.en.json

```diff
@@ -1116,29 +1116,21 @@
     "description": "Part of WebUSB error message"
   },
   "webusb-error-clear-connect-description-2": {
-    "defaultMessage": "Close any other tabs that may be using WebUSB (e.g. MakeCode, Python Editor), or unplug and replug the micro:bit before trying again.",
+    "defaultMessage": "Close any other tabs that may be using WebUSB (e.g. MakeCode, Python Editor, machine learning tool), or unplug and replug the micro:bit before trying again.",
```

Comment:
I'm not sure that we want to add the ML tool here but one to confirm on Monday.

Reply:
This is still unresolved, but I've simplified this back to being just MakeCode and the Python Editor for now.